CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 0.21.9
+
+**Commit Delta**: [Change from 0.21.8 release](https://github.com/plus3it/watchmaker/compare/0.21.8...0.21.9)
+
+**Released**: 2021.04.26
+
+**Summary**:
+
```

Comment:
Can we add a top-level bullet here about supporting Salt 3003? I'm also working on a PR to add a blurb about the salt support to the docs...

Reply:
Done